### ISSUE TITLE
Is it possible to load an image without reseting the center/zoom?

### ISSUE THREAD

**Issue by mwcraig** [ejeschke/ginga#786]
I've been playing around with blinking between images in astrowidgets. Right now I'm doing that by calling `load_fits` and then set the center and zoom_level. That leads to a flicker because the image is initially displayed centered and zoomed to fit the window and then quickly changes to the zoom/center that I want.

Example gif:

![Untitled](https://user-images.githubusercontent.com/1147167/62382262-2e79f600-b51b-11e9-97ad-bbae755838df.gif)

**Comment by mwcraig** [comment]
The notebook in the gif is here: https://gist.github.com/mwcraig/2ca84a5a29e61d5aa871b059505dd28a

**Comment by pllim** [comment]
Matt is going to try to see if the notebook can be made to use one image widget per image file, and then the blinking to switch out the widget instead of reloading the images in the same widget.

If the above does not work out, some kind of caching is necessary to avoid reloading on switch. In Ginga, it has `Datasrc` caching system, but that is not ideal for `astrowidget` that wants to interface with other viewer backends as well.

**Comment by ejeschke** [comment]
@mwcraig, the auto-centering, auto-zooming, and auto-levels when you switch between images can all be turned off. See [this section](https://ginga.readthedocs.io/en/stable/dev_manual/image_viewer.html#auto-configuration) of the docs for details.

**Comment by ejeschke** [comment]
> Matt is going to try to see if the notebook can be made to use one image widget per image file, and then the blinking to switch out the widget instead of reloading the images in the same widget.

That approach also works, and is basically the approach I took with the `Pan`, `Header` and `Info` plugins in order to speed performance when you switch channels in the reference viewer. However, it does make the code more complicated and use a bit more memory.  In #761, I refactored them to just reload a single viewer and I found the performance difference to be almost undetectable. However, I want to test a little more and maybe make some timing tests to make sure before we merge that.
